### ISSUE TITLE
Run gunicorn as a non-root user

### ISSUE THREAD

**Issue by RobHooper** [open-contracting/data-registry#166]
Processes in containers should not run as root, instead we should be creating a user and running `gunicorn` / `python` as this user.

This will improve system security by reducing python's access, see also [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).

We were alerted about from our rootkit monitoring script.

**Comment by jpmckinney** [comment]
That's basically this issue: https://github.com/open-contracting/deploy/issues/307

I know very little about Docker. We do have a `deployer` user, but when I run `docker-compose down; docker-compose up -d`, the process ends up running as root (because I think that's how the Docker daemon does things by default).

There are pages linked from that issue about running 'rootless' but I haven't read further.

If you can take this on, that'd be great.

Feel free to close in favor of the other issue (which can be broken up into smaller issues).

**Comment by yolile** [comment]
Indeed, this is another best practice that we should follow. This is about the user inside the container itself. That user is defined in the Dockerfile, normally by default is the root user, but in the Dockerfile you can create another user and then use the USER directive to star the container using that user instead of root, see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user

**Comment by jpmckinney** [comment]
Right - I'll move this to the deploy repo, as it affects all repos with Dockerfile's.
